jwst/datamodels/schemas/filteroffset.schema.yaml

```diff
@@ -8,6 +8,8 @@ allOf:
 - $ref: keyword_exptype.schema
 - $ref: keyword_filter.schema
```

Comment:
Do we need FILTER in the headers of these ref files? It isn't currently used as a file selector in CRDS for MIRI, and it sounds like it won't be used as a selector for NIRCam or NIRISS either. Doesn't hurt to have it there, but I don't think it's needed. It's only needed as a key in the ref file data itself.

Reply:
removed